### PR TITLE
Create maintenance page vue

### PR DESCRIPTION
**Pull request by gonzalo-plaza**

### CODE REVIEW

**Comment by gonzalo-plaza** [comment]
@Andresws12 I have change the styles appling the code review of @JuanJosePG , but I need do the colors palette branch befor of apply the variables in maintenance page vue.

File: src/views/Maintenancepage.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import mainStore from '@/store/main-store/MainStore';
+@Component({
+    name: 'Maintenance',
+    components: {},
+})
+export default class Maintenance extends Vue {
```
Comment (Andresws12) [inline src/views/Maintenancepage.vue]: @gonzalo-plaza There are two problems jejeje on the one hand, you are calling this component Maintenancepage.vue and all components are calling in UpperCamelCase, so this component should called MaintenancePage.vue, on the other hand, if this component is calling MaintenancePage why you declare the class Maintenance? and the name from the component Maintenance? The name from the component and the name should be same that the name from the file

File: src/views/Maintenancepage.vue [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
...CAtNzMuNTkpIi8+CiAgPHBhdGggZD0iTTM5NS4xOCAxMjAuMzZBMTAuMTkgMTAuMTkgMCAwMDM4NSAxMTAuMThoLTFhMjQuNzMgMjQuNzMgMCAwMC40Ni00Ljc0IDI1LjA1IDI1LjA1IDAgMDAtNDkuNTYtNS4yIDE1LjU4IDE1LjU4IDAgMTAtNS43NyAzMC4yM2g1NmExMC4xOCAxMC4xOCAwIDAwMTAuMDUtMTAuMTF6IiBjbGFzcz0iY2xzLTMiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC01OCAtNzMuNTkpIi8+CiAgPHBhdGggZD0iTTM0Ny41MSAxMDMuNjFhLjU1LjU1IDAgMDEuOTUgMGwzLjA3IDUuMzMgMy4wOCA1LjMzYS41NS41NSAwIDAxLS40Ny44MmgtMTIuMzFhLjU1LjU1IDAgMDEtLjQ4LS44MmwzLjA4LTUuMzN6IiBjbGFzcz0iY2xzLTIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC01OCAtNzMuNTkpIi8+CiAgPHBhdGggZD0iTTI4Ny43OSAzOS42MWg0LjcxdjI1LjAzaC00LjcxeiIgY2xhc3M9ImNscy0yIi8+CiAgPHBhdGggZD0iTTMwNi4xNCAyOS43NWg0LjcxdjI1LjAzaC00LjcxeiIgY2xhc3M9ImNscy00Ii8+CiAgPHBhdGggZD0iTTM2NS44NiAxMzhhLjU0LjU0IDAgMDAuOTQgMGwzLjA4LTUuMzMgMy4wOC01LjMzYS41NC41NCAwIDAwLS40Ny0uODJoLTEyLjMxYS41NS41NSAwIDAwLS40OC44MmwzLjA4IDUuMzN6IiBjbGFzcz0iY2xzLTQiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC01OCAtNzMuNTkpIi8+CiAgPHRleHQgZmlsbD0iIzMwMzg0MSIgZm9udC1mYW1pbHk9Ik9wZW5TYW5zLVNlbWlCb2xkLE9wZW4gU2FucyIgZm9udC1zaXplPSI0OCIgZm9udC13ZWlnaHQ9IjYwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMTMzLjQxIDE2OS4zKSI+CiAgICBQR0FKPHRzcGFuIHg9IjExMC40NiIgeT0iMCIgZm9udC1zaXplPSIxMiI+PC90c3Bhbj4KICA8L3RleHQ+Cjwvc3ZnPgo="
```
Comment (Andresws12) [inline src/views/Maintenancepage.vue]: @gonzalo-plaza Now we can add this image on assets/img, the base 64 is only to static page, please add this iamge in this dir and use this image

File: src/views/Maintenancepage.vue [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<template>
+    <main class="maintenance-page">
+        <div class="maintenance-page_content level is-mobile">
```
Comment (Andresws12) [inline src/views/Maintenancepage.vue]: @gonzalo-plaza Why are you using level and is-mobile, if you cadd add this styles using the class _content  and is not showing correctly on my laptop, please review in others desktop views
image

File: src/views/Maintenancepage.vue [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                    {{ $t('views.maintenance.subtitle') }}
+                </div>
+            </div>
+        </div>
+    </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import mainStore from '@/store/main-store/MainStore';
```
Comment (Andresws12) [inline src/views/Maintenancepage.vue]: @gonzalo-plaza Why are you calling the store? and on the code, you declare the store, and declare currentLanguaje, but... where are you using currentLanguaje? this get method onli return es or en, if the language is english or spanish, but you need change this languaje,  for this, we have a generic component called 'ButtonTranslation' on login dir from appointment system, maybe you cand copy and paste this component in common dir, and use this component like is using in login view from appointments

To use this component you can see how we use this componennt on MainLoginForm componentthere are a component

File: src/views/Maintenancepage.vue [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    &_title {
+        font-weight: bold;
+        font-size: 50px;
+        color: #3a4750;
+    }
+    &_subtitle {
+        font-size: 30px;
+    }
+}
+@media (max-width: 750px) {
```
Comment (Andresws12) [inline src/views/Maintenancepage.vue]: @gonzalo-plaza We have tree diferents mixins to use this media, you can use touch, mobile and desktop, like that Jose is using in appoinment maintenance page

**Review by Andresws12 — CHANGES_REQUESTED** [review]

File: src/views/MaintenancePage.vue [hunk 6]
```diff
@@ -0,0 +1,82 @@
+<template>
+    <main class="maintenance-page">
+        <div class="maintenance-page_content level">
+            <img
+                class="maintenance-page_img"
+                src="../assets/img/mantenimiento-image.svg"
```
Comment (Andresws12) [inline src/views/MaintenancePage.vue]: @gonzalo-plaza English please

File: src/views/MaintenancePage.vue [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+                alt="Maintenance Image"
+                title="Maintenance-Image"
+            />
+            <div class="maintenance-page_description level">
+                <div class="maintenance-page_title">
```
Comment (Andresws12) [inline src/views/MaintenancePage.vue]: @gonzalo-plaza It's a good practice use h1 to titles and h2 to subtitles, cause the SEO works veri good with h1 and h2, in maintenance is not importatn but.... if good use this

File: src/views/MaintenancePage.vue [hunk 8]
```diff
@@ -0,0 +1,82 @@
+<template>
+    <main class="maintenance-page">
```
Comment (Andresws12) [inline src/views/MaintenancePage.vue]: @gonzalo-plaza this component is not the main from the page, cause the main is app.vue, maybe is confusing if we add main on the views or components, cause the main represent the main content from the app, and this content is a one part from this,.

File: src/views/MaintenancePage.vue [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+                    {{ $t('views.maintenance.subtitle') }}
+                </div>
+            </div>
+            <button-translation></button-translation>
+        </div>
+    </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
```
Comment (Andresws12) [inline src/views/MaintenancePage.vue:24]: @gonzalo-plaza please add a white line to separe the imports, cause theese are diferents imports, on the one hand, we import a library and on the other hand we are importing the components and also add withe line space, to separe the imports component that '@components'

**Review by Andresws12 — CHANGES_REQUESTED** [review]

**Comment by gonzalo-plaza** [comment]
@Andresws12 This code can be review.

**Comment by Andresws12** [comment]
@gonzalo-plaza On my laptop the image is so big 
image

**Comment by gonzalo-plaza** [comment]
@Andresws12 Ok!! Y review more posibilitys to apply the responsive

**Pull request merged**